src/Sulu/Bundle/WebsiteBundle/Resources/config/analytics.xml

```diff
@@ -37,7 +37,7 @@
             <argument>%kernel.environment%</argument>
 
             <tag name="sulu.context" context="website"/>
-            <tag name="kernel.event_listener" event="kernel.response" method="onResponse"/>
+            <tag name="kernel.event_listener" event="kernel.response" method="onResponse" priority="-5"/>
```

Comment:
it has be called after the symfony response-listener which prepares the response and set the correct mime-type.